File: model/common/PocketObject.java
package model.common;

public class PocketObject {
  private final int id;
  private boolean isLost;

  public PocketObject(int id) {
    this.id = id;
  }

  public boolean isLost() {
    return isLost;
  }

  public void setLost(boolean lost) {
    isLost = lost;
  }

  public int getId() {
    return id;
  }
}

File: model/common/PocketObject.cs
namespace Model.Common
{
    public class PocketObject
    {
        private readonly int _id;
        private bool _isLost;

        public PocketObject(int id)
        {
            _id = id;
        }

        public bool IsLost
        {
            get { return _isLost; }
            set { _isLost = value; }
        }

        public int Id
        {
            get { return _id; }
        }
    }
}
